### PR TITLE
Add 'download Thunderbird' notice bar to Thunderbird landing page

### PR DESCRIPTION
**Pull request by mmmavis**
Closes https://github.com/MozillaFoundation/Thunderbird-Migration/issues/30

See: https://donate-wagtail-staging-pr-715.herokuapp.com/en-US/?tbdownload=true

### CODE REVIEW

**Comment by beccaklam** [comment]
@mmmavis I think the location of the banner works! The red makes it look an error message though and a little hard to read. Could we try grey (#f2f2f2) and use nunito for the typeface instead?

**Comment by mmmavis** [comment]
@beccaklam yup we can make those updates. I chose red just so that I could have **_a_** non-white background shown 😁

**Comment by beccaklam** [comment]
@mmmavis Thanks for implementing those changes! Does it look too subtle to you now though? 🤔

File: source/sass/components/_app.scss [hunk 1]
```diff
@@ -2,4 +2,14 @@
   &--rtl {
     direction: rtl;
   }
+
+  &__notice-bar {
+    background: $color--grey-05;
+    text-align: center;
+
```
Comment (beccaklam) [inline source/sass/components/_app.scss]: Can we also add some padding to the top and bottom here? 8px should be good!

**Comment by mmmavis** [comment]
@beccaklam 

> @mmmavis Thanks for implementing those changes! Does it look too subtle to you now though? 🤔

I'm not sure if we are restricted to the colours we have established on the site. If we are, I think `#f2f2f2` is probably our best choice among all...
image

**Comment by beccaklam** [comment]
@mmmavis That's true. When it shows up on the page could we have it animate slightly to draw more attention to it? If not, that's fine, we can ship it!

**Comment by mmmavis** [comment]
@beccaklam 

> @mmmavis That's true. When it shows up on the page could we have it animate slightly to draw more attention to it? If not, that's fine, we can ship it!

Yeah I was thinking about making it animated a bit but since we are aiming to have this in by Wednesday, animation will have to stay as nice-to-have for now.

Gonna assign people for code review now.

**Comment by mmmavis** [comment]
@alanmoo gonna assign you are code reviewer since you filed the original ticket and have context.

File: donate/thunderbird/templates/pages/core/landing_page.html [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+{% extends "pages/core/landing_page_master.html" %}
+
+{% block notice_bar %}
+{% if request.GET.tbdownload == 'true' %}
```
Comment (mmmavis) [inline donate/thunderbird/templates/pages/core/landing_page.html]: do we want `true` to be case-sensitive?

Comment (Pomax) [reply donate/thunderbird/templates/pages/core/landing_page.html]: Depends: is this a string or a boolean? Or, is the only thing we really care about whether that argument exists? E.g. `{% if 'tbdownload" in request.GET %} ... {% endif %}`

Comment (mmmavis) [reply donate/thunderbird/templates/pages/core/landing_page.html]: Forgot to include the link to the original [ticket](https://github.com/MozillaFoundation/Thunderbird-Migration/issues/30) (I have also updated PR description to include it).

> Depends: is this a string or a boolean? 

I'll have to defer this quesiton to @alanmoo

**Comment by mmmavis** [comment]
A few template files were changed quite a bit today. I'm gonna open a new PR instead since rebasing & fixing merge conflicts for this PR will take a lot longer than opening a new one.

File: donate/thunderbird/templates/pages/core/landing_page.html (hunk 2 above)
Comment (alanmoo) [reply donate/thunderbird/templates/pages/core/landing_page.html]: Just check if it exists.

**Comment by alanmoo** [comment]
> @mmmavis That's true. When it shows up on the page could we have it animate slightly to draw more attention to it? If not, that's fine, we can ship it!

I don't think that's necessary- this is the page that users are directed to after they start a download of Thunderbird, the main thing here is the ask for donation; let's not distract them with the animation.

File: donate/thunderbird/templates/pages/core/landing_page.html (hunk 2 above)
Comment (alanmoo) [reply donate/thunderbird/templates/pages/core/landing_page.html]: I thought I responded to this comment, but just check if the argument exists.

Comment (Pomax) [reply donate/thunderbird/templates/pages/core/landing_page.html]: ah, yeah in that case we can use `{% if "tbdownload" in request.GET %} ... {% endif %}`

**Comment by mmmavis** [comment]
Cool! I filed a [new PR](https://github.com/mozilla/donate-wagtail/pull/737) instead as I didn't wanna deal with not-so-straightward rebasing. Will tag people for quick reviw once review app is ready for 👀

**Pull request closed without merging**